modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -113,7 +99,33 @@ def generate_pdf # rubocop:disable Metrics/MethodLength
         end
 
         def synchronous
-          render json: {}
+          auto_claim = shared_submit_methods
+          # This kicks off the first of three jobs required to fully establish the claim
+          process_claim(auto_claim, false) unless Flipper.enabled? :claims_load_testing
```

Comment:
technically this could go in the `shared_submit_methods` as well and we could pass the boolean to that, but ? more of a nitpick.

Reply:
I need to think on the way this cascades errors through the jobs. I had not anticipated this. not a huge fan. I need to understand how this can impact us 🤔